src/k8s-extension/azext_k8s_extension/commands.py

```diff
@@ -20,6 +20,6 @@ def load_command_table(self, _):
                             is_preview=True) \
             as g:
         g.custom_command('create', 'create_k8s_extension')
-        g.custom_command('delete', 'delete_k8s_extension', confirmation=True)
+        g.custom_command('delete', 'delete_k8s_extension')
```

Comment:
I thought 'confirmation=True' will automatically add the prompt for confirmation, in case of delete.  What is the difference between how it was and how it is changed?  Can you explain?

Reply:
Never mind; I see why this change was done!